src/Symfony/Component/Notifier/Bridge/Bluesky/BlueskyTransport.php

```diff
@@ -65,19 +66,28 @@ protected function doSend(MessageInterface $message): SentMessage
         $post = [
             '$type' => 'app.bsky.feed.post',
             'text' => $message->getSubject(),
-            'createdAt' => (new \DateTimeImmutable())->format('Y-m-d\\TH:i:s.u\\Z'),
+            'createdAt' => \DateTimeImmutable::createFromFormat('U', time())->format('Y-m-d\\TH:i:s.u\\Z'),
```

Comment:
Is this just CS? Or what is the difference?

Reply:
I've made this change to use the `time` function that allow me to use the `ClockMock` in tests.